here-naksha-app-service/src/test/java/com/here/naksha/app/service/ReadFeaturesByRadiusTest.java

```diff
@@ -254,17 +274,81 @@ private static Stream<Arguments> standardTestParams() {
                     ),
                     "ReadFeatures/ByRadius/TC17_withRefFeatureMissingGeometry/feature_response_part.json",
                     404
+            ),
+            standardTestSpec(
```

Comment:
It is confusing not to find TC 18 and 19 in this list (from 17, it jumps to 20) ... just put one liner comment that they are implemented as separate functions